test/zoning-district-class/zoning-district-class.e2e-spec.ts

```diff
@@ -54,6 +60,61 @@ describe("Zoning District Classes e2e", () => {
     });
   });
 
+  describe("getZoningDistrictClassesById", () => {
+    it("should 200 and return a zoning district class by id", async () => {
+      const mock = zoningDistrictClassRepostoryMock.findByIdMocks[0];
```

Comment:
Another community service request. ` zoningDistrictClassRepostoryMock` is missing an i in `Repostory`

Reply:
lol, this was a subtle one to catch. The spelling error was duped all over the file